src/vmware/azext_vmware/_params.py

```diff
@@ -134,3 +134,11 @@ def load_arguments(self, _):
         c.argument('out', help='Standard output stream from the powershell execution.')
         c.argument('named_outputs', action=ScriptExecutionNamedOutputAction, nargs='*', help='User-defined dictionary.')
         c.argument('script_cmdlet_id', help='A reference to the script cmdlet resource if user is running a AVS script.')
+
+    with self.argument_context('vmware workload-network dhcp') as c:

```

Comment:
I think it would be more clear if you added a `vmware workload-network dhcp server create` and a `vmware workload-network dhcp relay create` argument contexts and moved the specific arguments.

Reply:
The `create` and `update` commands have the same parameters, so I put them together for `relay` and `server`.

Reply:
Okay, but what about splitting up `dhcp server` from `dhcp relay`?